src/main/webapp/js/finscholar/init.js

```diff
@@ -30,22 +30,14 @@ const init = async () => {
   // Render the college page by loading in the data and rendering the 
   // associated soy template.
   try {
-    await renderCollege(GoogDom.getElement('college'));
+    const collegePage = new CollegePageView(GoogDom.getElement('college'));
+    await collegePage.renderCollege();
   } catch(err) {
     alert('Failed to initialize page. College page could not be rendered.');
     console.log(err);
   }
 };
 
-/**
- * Render the college page.
- * @param {Element} element - The DOM element to attach the college page to.
- */
-const renderCollege = async (element) => {
-  const collegePage = new CollegePageView(element);
-  await collegePage.renderPage();
-};
-
 window['onload'] = init;
```

Comment:
I'm curious to know if it still works if you remove this line. I think this one and the one below it are functionally equivalent.

Reply:
I didn't write this part so I don't know lol.

Reply:
Hmm, don't worry about it then. We can make a new PR later after we all merge. It's possible that I left it in by accident. :)

Reply:
So it did work without that line. Nice!